src/gtk/textctrl.cpp

```diff
@@ -796,9 +819,6 @@ bool wxTextCtrl::Create( wxWindow *parent,
         // work around probable bug in GTK+ 2.18 when calling WriteText on a
         // new, empty control, see https://github.com/wxWidgets/wxWidgets/issues/11409
         gtk_entry_get_text((GtkEntry*)m_text);
-
-        if (style & wxNO_BORDER)
-            gtk_entry_set_has_frame((GtkEntry*)m_text, FALSE);
```

Comment:
Now the diff shows this as removed. I think only the `#ifndef __WXGTK3__` should have been removed, and the rest restored to how it originally was. Unless there is a reason why this should be removed, or not used with GTK3.

Reply:
This absolutely should be restored, no idea why was it removed.